### DISCUSSION TITLE
Open for local network PCs?

### DISCUSSION THREAD

**Discussion by aaronbychen** [open-webui/open-webui#3779] [General]
Hi, I'm using windows and running ollama on wsl and open webui on docker of wsl. Now http://localhost:8080/ works fine.

I installed using this command on wsl "docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:main"

Is it possible that I can expose a port so other PCs that connect to the same local network can access to open webui? I've tried for a long time and it's a bit confusing for me. Thank you!

**Comment by eguar1978** [comment]
I have the same issue. This is how I create the container:

docker run -d --name open-webui -p 3000:8080 -p 11434:11434 -v open-webui:/app/backend/data -e USE_OLLAMA_DOCKER=true -e OLLAMA_BASE_URL=http://0.0.0.0:11434 ghcr.io/open-webui/open-webui:ollama bash start.sh

And this is how the API is created for the Docker local network only:

Proto Recv-Q Send-Q Local Address           Foreign Address         State
tcp        0      0 127.0.0.1:11434         0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:8080            0.0.0.0:*               LISTEN
root@0f395443da11:/app/backend#
